parser/src/parser/file.rs

```diff
@@ -40,17 +40,25 @@ impl ParserContext<'_> {
                     import_statements.push(self.parse_import_statement()?);
                 }
                 Token::Circuit => {
-                    self.expect(Token::Circuit)?;
-                    let (id, circuit) = self.parse_circuit()?;
-                    circuits.insert(id, circuit);
+                    let span = self.expect(Token::Circuit)?;
+                    self.emit_err(ParserError::cricuit_definition(&span));
+
+                    let (id, structure) = self.parse_struct()?;
+                    structs.insert(id, structure);
+                }
+                Token::Struct => {
+                    self.expect(Token::Struct)?;
+
+                    let (id, structure) = self.parse_struct()?;
+                    structs.insert(id, structure);
                 }
                 Token::Ident(ident) => match ident.as_ref() {
                     "test" => return Err(ParserError::test_function(&token.span).into()),
                     kw @ ("struct" | "class") => {
```

Comment:
```suggestion
                    kw @ ("circuit" | "class") => {
```
You can now remove the `circuit` keyword and token.